src/script/components/manifest-options.ts

```diff
@@ -489,6 +489,21 @@ export class AppManifest extends LitElement {
     super.disconnectedCallback();
   }
 
+  renderDownloadButton(): TemplateResult {
+    if(this.iconsList) {
+     return html`<loading-button
+            class="hidden-sm"
+            appearance="outline"
+            ?loading=${this.awaitRequest}
+            @click=${this.downloadIcons}
+            >${localeStrings.button.download}</loading-button
+          >`
+    } else {
```

Comment:
Since the if branch returns, we don't need an else branch. Instead, we can just return html`` at the end of the function.